cost/cost_test.go

```diff
@@ -0,0 +1,45 @@
+package cost_test
+
+import (
+	"testing"
+
+	"github.com/cycloidio/terracost/cost"
+	"github.com/shopspring/decimal"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewHourly(t *testing.T) {
+	c := cost.NewHourly(decimal.NewFromFloat(1.23))
+	assertDecimalEqual(t, decimal.NewFromFloat(897.9), c.Decimal) // 897.9 = 1.23 * 730
+}
+
+func TestNewMonthly(t *testing.T) {
+	c := cost.NewMonthly(decimal.NewFromFloat(1.23))
+	assertDecimalEqual(t, decimal.NewFromFloat(1.23), c.Decimal)
+}
+
+func TestCost_Hourly(t *testing.T) {
+	c := cost.NewMonthly(decimal.NewFromFloat(897.9))
+	assertDecimalEqual(t, decimal.NewFromFloat(1.23), c.Hourly())
+}
+
+func TestCost_Monthly(t *testing.T) {
+	c := cost.NewHourly(decimal.NewFromFloat(1.23))
+	assertDecimalEqual(t, decimal.NewFromFloat(897.9), c.Monthly()) // 897.9 = 1.23 * 730
```

Comment:
Maybe instead of comment what is it declare a constant to reuse?

Reply:
Using the `cost.HoursPerMonth` value instead